content/blog/2017/2017-01-12-declarative-pipeline-beta-2.adoc

```diff
@@ -0,0 +1,122 @@
+You can read more about Declarative Pipeline
+link:/blog/2016/12/19/declarative-pipeline-beta/[in the blog post introducing the first beta]
+from December, but we wanted to update you all on the syntax changes in the
+second beta. These syntax changes are the last compatibility-breaking changes to
+the syntax for the 1.0 release planned for February, so you can safely start
+using the 0.8.1 syntax now without needing to change it when 1.0 is released.
+
+== Syntax Changes
+
+=== Agent
+In order to support more detailed and clear configuration of agents, as well as
+making agent syntax more consistent with the rest of the Declarative Pipeline
+syntax, we've moved the agent configuration into blocks. The `agent any` and
+`agent none` configurations work the same as previously, but `label`, `docker`
+and `dockerfile` now look like the following:
+
+[pipeline]
+----
+// Just specifying a label is simple.
+agent {
+    label "some-label"
+}
+
+// If you're just specifying a Docker image, you can use this simple syntax.
+agent {
+    docker "ubuntu:lts"
+}
+
+// When you are specifying a label or other arguments, docker looks like this:
+agent {
+    docker {
+        image "ubuntu:lts"
+        label "docker-label"
+        args "-v /tmp:/tmp -p 8000:8000"
+    }
+}
+
+// When you're building an image from "Dockerfile" in your repository and
+// don't care what node is used or have additional arguments, you can again
+// use a simple syntax.
+agent {
+    dockerfile true
+}
+
+// When you're building an image from a differnt file, or have a label or other
+// arguments, use the following syntax:
+agent {
+    dockerfile {
+        filename "OtherDockerfile"
+        label "docker-label"
+        args "-v /tmp:/tmp -p 8000:8000"
+    }
+}
+----
+
+=== `when` - conditional stage execution
+We introduced the `when` section a couple releases ago, but have made some
+changes to its syntax here in 0.8.1. We wanted to add some simpler ways to
+specify common conditions, and that required we re-work the syntax accordingly.
+Here's what `when` looks like now:
+
+[pipeline]
+----
+// One of the most common conditions is running a stage only if you're on a
+// specific branch. You can also use wildcards like "*/master".
+when {
+    branch "master"
+}
+
+// Another built-in condition is the environment condition, which checks to see
+// if a given environment variable is set to a given value.
+when {
+    environment name: "SOME_ENV_VAR", value: "SOME_VALUE"
+}
+
+// Lastly, there's the expression condition, which resolves an arbitrary
+// Pipeline expression. If the return value of that expression isn't false or
+// null, the stage will execute.
+when {
+    expression {
+        echo "Should I run?"
+        return "foo" == "bar"
+    }
+}
+----
+
+=== Properties, options and wrappers
+We've renamed the old `properties` section to `options`. The `options` section
+is now where you'll put job properties, like `buildDiscarder`, Declarative-specific
```

Comment:
"The `options` section is now where you'll put job properties" is a confusing statement, and uses the term "job" which I have been trying to purge from al references related to Pipeline.

Suggest: "The `options` section is now where you will put Pipeline options such as foobar, etc"